workers/generic-worker/usage.go

```diff
@@ -203,7 +201,7 @@ and reports back results to the queue.
           taskclusterProxyPort              Port number for taskcluster-proxy HTTP requests.
                                             [default: 80]
           tasksDir                          The location where task directories should be
-                                            created on the worker. [default: ` + fmt.Sprintf("%q", defaultTasksDir()) + `]
+                                            created on the worker. [default varies by platform]
```

Comment:
I guess this was necessary to generate the help text? I'm sad to lose the information (although, to be fair, I've never used it). Maybe some thing that matches and replaces `created on the worker. [default:.*`, similar to the first line replacement?

Reply:
My thinking was that the information in the README was incorrect (and I have in the past wondered why it uses `/Users` on linux.. turns out it doesn't).  But it's a good point that the `--help` output can be platform-specific.  Let me see what I can do..

Reply:
(I only noticed this because running `refresh_readme.sh` locally changed `/Users` to `/home`)